### PR TITLE
Added method to click iOS Alert button in Appium v2

### PR DESCRIPTION
**Pull request by zastress**
## Implements

Handle iOS Alert interraction in Appium v2 starting with xcuitest driver v6

### CODE REVIEW

**Comment by ssharma0312** [comment]
@zastress which release would this PR will go in?

**Comment by zastress** [comment]
I have no idea, i am not a maintainer of the repo. I am also waiting for an answer

**Review by serhatbolsu — APPROVED** [review]

**Comment by serhatbolsu** [comment]
Thank you @zastress

**Pull request merged**
